src/app/components/ListLives.tsx

```diff
@@ -0,0 +1,175 @@
+    font-weight: bold;
+    font-size: 14px;
+    padding: 5px;
+`;
+
+const StyledValue = styled.div`
+    font-size: 16px;
+    padding: 5px;
+`;
+
+const StyledRow = styled(Row)`
+    padding: 10px;
+    border: 1px solid #ccc;
+    border-radius: 10px;
+    .mainDetails {
+        height: 100%;
+        background-color: #f5f5f5;
+        padding: 1em;
+        margin-right: 10px;
+        border-radius: 5px;
+    }
+
+    .otherDetails {
+        height: 100%;
+        padding: 1em;
+        margin-right: 10px;
+    }
+
+    .actions {
+        display: flex;
+        align-items: center;
+    }
+`;
+
+const StyledField = styled('div')<FieldLayoutProps>`
+    display: flex;
+    flex-flow: ${props => props.layout} nowrap;
+
+    .label {
+        width: ${props => (props.layout === 'row' ? '50%' : '100%')};
+    }
+    .value {
+        width: ${props => (props.layout === 'row' ? '50%' : '100%')};
+    }
+`;
+
+// React Components
+// ------------------------------------------------------
+
+const Field = ({ label, value, layout }: FieldProps & FieldLayoutProps): JSX.Element => {
+    const valueIsArray = Array.isArray(value);
+
+    return (
+        <StyledField layout={layout}>
+            <StyledLabel className="label">{label}</StyledLabel>
+            <StyledValue className="value">
+                {!valueIsArray && value}
+                {valueIsArray && (
+                    <ul>
+                        {value.map(val => (
+                            <li key={val}>{val}</li>
+                        ))}
+                    </ul>
+                )}
+            </StyledValue>
+        </StyledField>
+    );
+};
+
+const ListLives = () => {
+    const { t } = useTranslation(['life']);
+    const history = useHistory();
+    const { data, loading } = useGetListLivesQuery({
+        fetchPolicy: 'network-only',
+        nextFetchPolicy: 'cache-first',
+    });
+    const listLives = data?.listLives || [];
+    const noLives = !listLives?.length;
+
+    if (!loading && noLives) {
+        return (
+            <Empty description={t('life:noLives')}>
+                <Button onClick={() => history.push(`/lives/create`)} shape="round" type="primary">
+                    {t('life:listLives:createLife')}
+                </Button>
+            </Empty>
+        );
+    }
+
+    return (
+        <div className="listLivesWrapper">
+            <Button
+                id="createLife"
+                onClick={() => history.push(`/lives/create`)}
+                shape="round"
+                size="large"
+                type="primary"
+            >
+                {t('life:listLives:createLife')}
+            </Button>
+
+            <Divider />
+            <Skeleton loading={loading} active>
+                {listLives.map(life => (
+                    <React.Fragment key={life.id}>
+                        <StyledRow className="lifeRecordWrapper">
+                            <Col lg={6} sm={24}>
```

Comment:
Might be simpler to implement if we are to use antd table for this

Reply:
Yes, correct. If user provides bigger content for  description and hobbies fields then Table is not suitable. So that way i have created custom once.